pkg/rehearse/configmaps_test.go

```diff
@@ -0,0 +1,199 @@
+package rehearse
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+
+	prowconfig "k8s.io/test-infra/prow/config"
+
+	v1 "k8s.io/api/core/v1"
+
+	"k8s.io/client-go/kubernetes/fake"
+
+	"k8s.io/apimachinery/pkg/api/equality"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/diff"
+)
+
+var dryFalse = false
+var dryTrue = true
+var testNamespace = "test-namespace"
+var testRepo = "organization/project"
+var testPrNumber = 1234
+var testConfigDir = "ci-operator/configs"
+
+func createPresubmitWithEnv(env []v1.EnvVar) *prowconfig.Presubmit {
+	return &prowconfig.Presubmit{
+		JobBase: prowconfig.JobBase{
+			Spec: &v1.PodSpec{Containers: []v1.Container{{Env: env}}},
+		},
+	}
+}
+
+func makeCMKeyRef(key, name string) *v1.ConfigMapKeySelector {
+	return &v1.ConfigMapKeySelector{
+		LocalObjectReference: v1.LocalObjectReference{Name: name},
+		Key:                  key,
+	}
+}
+
+type FakeConfigFilesReader struct {
+	files map[string][]byte
+}
+
+func (f *FakeConfigFilesReader) Read(path string) (string, error) {
+	content, ok := f.files[path]
+	if ok {
+		return string(content), nil
+	}
+
+	return "", fmt.Errorf("Failed to read a file %s", path)
+}
+
+func TestCreate(t *testing.T) {
+	testLogger := logrus.New()
+	testLogger.SetOutput(ioutil.Discard)
```

Comment:
Seeing logs from tests is often useful

Reply:
https://github.com/openshift/ci-operator-prowgen/pull/49/commits/d8f71b4ffd2d83717659ce733846c3519d521e86